phase-4-explainer.md

```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
     readonly attribute unsigned long? rowSpan;
     readonly attribute long? setSize;
 
+    readonly attribute float? valueNow;
+    readonly attribute float? valueMin;
+    readonly attribute float? valueMax;
+
+    readonly attribute DOMString? autocomplete;
+    readonly attribute DOMString? checked;
+    readonly attribute DOMString? keyShortcuts;
+    readonly attribute DOMString? name;
+    readonly attribute DOMString? placeholder;
+    readonly attribute DOMString? role;
+    readonly attribute DOMString? roleDescription;
+    readonly attribute DOMString? valueText;
+
     readonly attribute ComputedAccessibleNode? parent;
     readonly attribute ComputedAccessibleNode? firstChild;
     readonly attribute ComputedAccessibleNode? lastChild;
     readonly attribute ComputedAccessibleNode? previousSibling;
     readonly attribute ComputedAccessibleNode? nextSibling;
 
-    readonly attribute boolean? atomic;
-    readonly attribute boolean? busy;
-    readonly attribute boolean? disabled;
-    readonly attribute boolean? modal;
-    readonly attribute boolean? readOnly;
-
-    readonly attribute float? valueNow;
-    readonly attribute float? valueMin;
-    readonly attribute float? valueMax;
-
     [CallWith=ScriptState] Promise ensureUpToDate();
 };
 ```
 
-TODO(meredithl): insert link to full list of potential properties in spec.
-
+See the [proposed Phase 4 Spec](spec/phase4.html) for a full list of properties available to be implemented.
```

Comment:
This should be a link to probably the hosted spec I imagine?

Reply:
This is fine for now.